modules/rosa-sdpolicy-am-aws-compute-types.adoc

```diff
@@ -354,11 +365,31 @@ Support for the GPU instance type software stack is provided by AWS. Ensure that
 - c6id.16xlarge (64 vCPU, 128 GiB)
 - c6id.24xlarge (96 vCPU, 192 GiB)
 - c6id.32xlarge (128 vCPU, 256 GiB)
```

Comment:
The instance-types from this line to line 380 (m5zn.xlarge) are listed on same line, 
may i know if those are kept in one line intentionally, if not i think we should use a new line for each type to make it easy to read.

Reply:
Good catch @AkashKanni! This was not intended. Should be fixed now.